inst/examples/germany_missing_report_dates.R

```diff
@@ -0,0 +1,65 @@
+# Load packages
+library(epinowcast)
+library(data.table)
+
+# Set cmdstan path
+cmdstanr::set_cmdstan_path()
+
+# Use 2 cores
+options(mc.cores = 2)
+
+# Load and filter germany hospitalisations
+nat_germany_hosp <- germany_covid19_hosp[location == "DE"][age_group %in% "00+"]
+nat_germany_hosp <- enw_filter_report_dates(
+  nat_germany_hosp,
+  latest_date = "2021-10-01"
+)
+
+# Make sure observations are complete
+nat_germany_hosp <- enw_complete_dates(
+  nat_germany_hosp,
+  by = c("location", "age_group")
+)
```

Comment:
Oh, one more thing I wanted to ask: if the user forgets to apply `enw_complete_dates()` in the preprocessing stage, and then wants to use the flagging of observations and likelihood skipping, do we get a problem?

Reply:
I think they can use the helper functions to do this but if they have non-consecutive data then that will be a problem. However, I think the timestep checker function should error if that is the case (and hopefully there is a unit test to check that..)